src/main/java/org/alliancegenome/curation_api/dao/slotAnnotations/constructSlotAnnotations/ConstructComponentSlotAnnotationDAO.java

```diff
@@ -19,7 +19,7 @@ protected ConstructComponentSlotAnnotationDAO() {
 	}
 
 	public void deleteAttachedNote(Long id) {
-		Query jpqlQuery = entityManager.createNativeQuery("DELETE FROM constructcomponentslotannotation_note WHERE relatednotes_id = '" + id + "'");
+		Query jpqlQuery = entityManager.createNativeQuery("DELETE FROM slotannotation_note WHERE relatednotes_id = '" + id + "'");
```

Comment:
@markquintontulloch the draw back to this kind of query is the hardcoding the table. I wondering if we could to a JSQL query. But for now.

Reply:
I feel like we should be able to do this with orphanRemoval = true and not need this at all

Reply:
Ya ... I agree.